spec/ruby/optional/capi/bignum_spec.rb

```diff
@@ -117,6 +117,67 @@ def ensure_bignum(n)
       val = @s.rb_big_pack(@max_ulong)
       val.should == @max_ulong
     end
+
+    ruby_version_is "2.2" do
+      platform_is wordsize: 64 do
+        it "packs max_ulong into _two_ ulongs to allow sign bit" do
+          val = @s.rb_big_pack_length(@max_ulong)
+          val.should == 2
+          val = @s.rb_big_pack_array(@max_ulong, 2)
+          val[0].should == @max_ulong
+          val[1].should == 0
+        end
+
+        it "packs A 72 bit positive Bignum into 2 unsigned longs" do
+          num = 2 ** 71
+          val = @s.rb_big_pack_length(num)
+          val.should == 2
+        end
+
+        it "packs A 2 ** 71 + 1 positive Bignum into 2 unsigned longs of 1 and 0x80" do
+          num = 2 ** 71 + 1
+          val = @s.rb_big_pack_array(num, 2)
+          val[0].should == 1;
+          val[1].should == 0x80;
+        end
+
+        it "packs A 2 ** 71 + 1 bit negative Bignum into 2 unsigned longs of 1 and ulongmax - 0xff + 0x80" do
```

Comment:
Avoid too much computations in the spec description, also it does not match what's in the code below.